playwright/_impl/_network.py

```diff
@@ -255,20 +315,54 @@ async def continue_(
         headers: Dict[str, str] = None,
         postData: Union[str, bytes] = None,
     ) -> None:
-        overrides: ContinueParameters = {}
+        # FIXME: This is getting triggered when it shouldn't be
```

Comment:
whats going on here?

Reply:
🙃 TBD!

Reply:
Fixed! This was prior to the greenlet fix.